README.md

```diff
@@ -80,7 +80,7 @@ wrapper classes or even your ENC (if it supports param classes). For example:
     # Agent and cron (or daemon):
     class { '::puppet': runmode => 'cron' }
```

Comment:
Unrelated, but I should do a follow up that also removes `::`. That was a Puppet 3/4 compatibility thing we no longer need to care about.